src/__tests__/ai-service.test.ts

```
@@ -0,0 +1,50 @@
+    }
+  });
+
+  afterEach(() => {
+    if (fs.existsSync('.function-indexer')) {
+      fs.rmSync('.function-indexer', { recursive: true });
+    }
+  });
+
+  describe('constructor', () => {
+    test('should throw error if no API key provided', () => {
+      delete process.env.OPENAI_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)